pkg/apis/apps/v1alpha1/description.go

```diff
@@ -70,6 +71,21 @@ type DescriptionStatus struct {
 	// Reason indicates the reason of DescriptionPhase
 	// +optional
 	Reason string `json:"reason,omitempty"`
+
+	// ManifestStatuses contains running status of manifests in spec.
+	// +optional
+	ManifestStatuses []ManifestStatus `json:"manifestStatus,omitempty"`
+}
+
+type ManifestStatus struct {
+	// feed
+	// +required
+	Feed Feed `json:"feed"`
+
+	// Status reflects running status of current manifest.
+	// +optional
+	// +kubebuilder:pruning:PreserveUnknownFields
+	Status runtime.RawExtension `json:"status,omitempty"`
```

Comment:
Try using another name.

It is weird to see `status.manifestStatus[0].status`.

Reply:
How about `feedStatus` ?